Flaky Test: Otter `StartFromState` test fails with specific seed in Turtle mode

The `StartFromState` Otter test fails when run in the Turtle environment with 3 nodes and seed `8777745741341107406L`. All three nodes go into `CHECKING` for a short time when they should not, and a coin round occurs:

```
2026-02-04 17:51:26.670 [pool-70-thread-3] [ERROR] WARN  org.hiero.consensus.hashgraph.impl.consensus.ConsensusImpl - Coin round 62, voting witness: (CR:1 H:20ca0c6fd333 BR:62)(CR:1 H:06ff67b8f51a BR:62)(CR:0 H:67aae59c5be9 BR:62) 
```

--- a/platform-sdk/consensus-otter-tests/src/testOtter/java/org/hiero/otter/test/StartFromStateTest.java
+++ b/platform-sdk/consensus-otter-tests/src/testOtter/java/org/hiero/otter/test/StartFromStateTest.java
@@ -24,6 +24,7 @@ import org.hiero.base.crypto.DetRandomProvider;
 import org.hiero.consensus.crypto.KeyGeneratingException;
 import org.hiero.consensus.crypto.KeysAndCertsGenerator;
 import org.hiero.consensus.crypto.SigningSchema;
+import org.hiero.consensus.event.creator.config.EventCreationConfig_;
 import org.hiero.otter.fixtures.Network;
 import org.hiero.otter.fixtures.Node;
 import org.hiero.otter.fixtures.OtterTest;
@@ -63,6 +64,12 @@ public class StartFromStateTest {
 
         // Setup simulation
         network.addNodes(numberOfNodes);
+        // The increase in event creation rate is to fix a past issue.
+        // This test encountered a coin round, it took many voting rounds to reach consensus. Because the checking
+        // status gets activated if an event does not reach consensus within a certain amount of time, the test would
+        // fail. By increasing the event creation rate, we can ensure that the network can create enough events to reach
+        // consensus in a timely manner.
+        network.withConfigValue(EventCreationConfig_.MAX_CREATION_RATE, 40);
         network.savedStateDirectory(Path.of("previous-version-state"));
         network.version(
                 currentVersion.copyBuilder().minor(currentVersion.minor()).build());

